recipe/meta.yaml

```diff
@@ -19,20 +19,26 @@ build:
     - python
 
 requirements:
+  build:
+    run_exports: # [osx and x86_64]
```

Comment:
run_exports should go under the main build section and not the one under requirements.